src/assets/google-script/google-script.ts

```
@@ -1,51 +1,150 @@
-      if (this.prevLang !== lang) {
-        initMap.remove();
+  private _removeGoogleScriptWithDelay(): Promise<void> {
+    return new Promise((resolve) => {
+      const existingScript = document.getElementById('google-maps-script');
+      if (existingScript) {
+        setTimeout(() => {
+          existingScript.remove();
+
+          try {
+            if (window?.google) {
+              delete window.google;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)